gen/meta/fvBD.json

```diff
@@ -2046,7 +2046,7 @@
                     "isConfigurable": true,
                     "propGlobalId": "34472",
                     "propLocalId": "8158",
-                    "label": "Optimize Wan Bandwidth between sites",
+                    "label": "Optimize Wan Bandwidth Between Sites",
```

Comment:
Wan -> WAN

Reply:
please check the file and re-assess if we should change this

Reply:
not required